week-2-task/salesByMonth.js

```diff
@@ -0,0 +1,15 @@
+// Calculate sales by month
+
+import salesData from "./json/salesData.json" assert { type: "json" };
+
+function salesByMonth(data, year, month) {
+  return data.filter((record) => {
+    const dates = new Date(record.date);
+    return dates.getMonth() + 1 === month && dates.getFullYear() === year;
+  });
+}
+
+const year = 2023;
```

Comment:
Provide input in ISO format and proceed with the task For eg `2024-08-15T03:48:51.598Z`